docs/source/version_control/diff.rst

```diff
@@ -0,0 +1,86 @@
+######
+ Diff
+######
+
+TensorBay supports showing changes between commits or drafts.
+
+Before operating the :ref:`reference/glossary:diff`, a dataset client instance with existing commit is needed.
+See more details in :ref:`version_control/draft_and_commit:Draft and Commit`
+
+**********
+ Get Diff
+**********
+
+TensorBay SDK supports getting the dataset :ref:`reference/glossary:diff`
+through :ref:`reference/glossary:basehead`. But now, only diffing
+between head and its parent commit is recommended.
+
+The data type of head hints the version status- string for commit, int for draft.
```

Comment:
`data type` -> `type`?
Is it better to use double back quotes around variables and types?